src/components/WwwFrame/LendMenu/TheLendMenu.vue

```diff
@@ -73,15 +78,19 @@ export default {
 				'Asia',
 				'Oceania'
 			],
-			loadingSemaphore: 0,
 			isRegionsLoading: true,
 			isChannelsLoading: true,
+			isUserDataLoading: false,
 			showMGUpsellLink: false,
 		};
 	},
 	apollo: {
 		query: pageQuery,
 		preFetch: true,
+		shouldPreFetch(config, { renderConfig }) {
```

Comment:
Do we need to remember to use this on all components now that prefetch? And is this just for options API components?

Reply:
Apollo plugin still only works for the options api. We could probably refactor the functionality so that it could be provided with a composable as well for the composition api.

Reply:
The only thing that was prefetched in this component was the user id. Before the lend menu is opened, the user id is only used to conditionally render the "My Kiva" sections of the mega menu and the list menu. I've changed those menus to also use the isUserDataLoading flag to show those sections, so that the user id is not required for the server render at all, and this prefetch can be skipped.